tenant/sunbird/newLandingPage.html

```diff
@@ -0,0 +1,919 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome: National Urban Learning Platforms</title>
+    <link rel="icon" href="">
```

Comment:
@Alfaj2302 rename the original index.html file and add these changes to the index.html

Reply:
we are still working on responsive once done will rename to index. currently it will new landing page

Reply:
ok